### ISSUE TITLE
unrecognized  ingredient - boron

### ISSUE THREAD

**Issue by cyberbofh** [msarilar/EDEngineer#312]
found unrecognized  ingredient - boron

using EDEngineer v1.0.9.1

**Comment by msarilar** [comment]
thanks for reporting !
do you have more details about the ingredient? e.g. it's rarity and category?
Here's an example of the information that I need for each ingredient (this is Polonium):

```
  {
    "Name": "Polonium",
    "Rarity": "VeryRare",
    "FormattedName": "polonium",
    "Kind": "Material",
    "Subkind": "Raw",
    "OriginDetails": [ "Surface prospecting", "Mission reward" ],
    "Group": "Category5"
  }
```

**Comment by JohnXaverius** [comment]
Hi,here is data:
{
    "Name": "Boron",
    "Rarity": "Standard",
    "FormattedName": "boron",
    "Kind": "Material",
    "Subkind": "Raw",
    "OriginDetails": [ "Minning" ],
    "Group": "Category7"
  }
![born lead rhenium](https://user-images.githubusercontent.com/29731838/36945354-e81fda6c-1fac-11e8-9b0a-d6a19d91f040.jpg)

**Comment by msarilar** [comment]
fixed in 8845f903bb8f2a9ac42d14ab74ab4cf6f855581d (will be in next update)
